### DISCUSSION TITLE
Spillway defaults

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#552] [Nixie]
I have a Nixie Shared Body pool.  I need the default position for the Spillway to be ON, unless turned off.  I also would like a reverse egg timer of sorts, where you can only turn the spillway off for so long before it turns back on.  This will ensure that the Spa has circulation and is being sanitized and not accidentally left off.

**Comment by rstrouse** [comment]
I run my spillway for a half hour every day through the schedules.   This is more than enough time to ensure the spa water gets filled with new water and it does not cause my pH to rise because the water is shedding carbonate.

How long do you want to allow the spillway to be off before it is forced back on?

**Reply by orrious** [reply]
I use my spillway differently I guess.  I leave it on all the time, however I have the valve set where it just trickles water over the large spillway normally.  Turning on the Waterfalls is what pushes all the flow over the spillway for effect.  

I wouldn't want the spillway off for more than ~4 hrs.  The only scenario I could think of where I would want it OFF was if I heated the Spa to one set point and then wanted to heat the Pool to another.

Why else does one turn off the Spillway?

**Reply by rstrouse** [reply]
> Why else does one turn off the Spillway?

Because it will push your pH higher (by a lot).  I used to think as you do and ran my spillway all the time fighting the calcium buildup and dealing the the constant pH rise.  These days it simply increases the acid demand in REM Chem.  On days I run it a lot it will increase the acid demand by 3x.

Bear in mind you can do what you desire and force the spillway back on after it has been off for 4 hours using the ruleProcessor if you want.  Nixie keeps track of the time that a circuit went off and when it was turned on.

**Reply by orrious** [reply]
Now you've got me curious, what causes the pH to rise?  I'm going to have to try turning it on for a couple hours a day.  I was worried about it not getting enough chlorine if I didn't run it.

**Reply by rstrouse** [reply]
When you aerate the water it will outgas CO2.  Part of this CO2 is in the form of carbonic acid H2CO3 and when CO2 outgasses it leave OH- behind.

**Reply by orrious** [reply]
That may be why I haven’t noticed it. When I say the spillway trickles, I mean just that. It doesn’t even disturb the pool water. 

The granite transition from my spa to pool is bullnosed, against my direction. So instead of the water hitting a sharp edge at the bottom and falling, it rides the granite and stone all the way into the pool without disturbing it.

It sounds like I could really increase the flow of the spillway and only run it 30 min a day
